pkg/lang/ir/conda.go

```diff
@@ -39,7 +39,7 @@ const (
 var (
 	// this file can be used by both conda and mamba
 	// https://mamba.readthedocs.io/en/latest/user_guide/configuration.html#multiple-rc-files
-	condarc = fileutil.EnvdHomeDir(".condarc")
+	condarc = "/opt/conda/.condarc"
```

Comment:
lgtm

/cc @kemingy

Reply:
Will this override https://github.com/tensorchord/envd/blob/d87afda9213fcb334ca55d247ba3c0663b7b180b/pkg/lang/ir/install-conda.sh#L24

But I guess it's okay?